wp-job-manager-template.php

```diff
@@ -410,6 +410,11 @@ function wpjm_get_job_listing_structured_data( $post = null ) {
 		$data['jobLocation']            = [];
 		$data['jobLocation']['@type']   = 'Place';
 		$data['jobLocation']['address'] = wpjm_get_job_listing_location_structured_data( $post );
+		if ( $post->_remote_position ) {
+			$data['additionalProperty']          = array();
+			$data['additionalProperty']['@type'] = 'PropertyValue';
+			$data['additionalProperty']['value'] = 'TELECOMMUTE';
```

Comment:
I think `jobLocationProperty` suits better here. Can we update this part to use that instead?
```suggestion
			$data['jobLocationType'] = 'TELECOMMUTE';
```

Reply:
Nice